package.json

```diff
@@ -32,7 +32,7 @@
     "generate-notices": "generate-license-file --input package.json --output THIRD-PARTY-NOTICES --overwrite"
   },
   "dependencies": {
-    "@yext/search-headless": "^2.0.0-alpha.133",
+    "@yext/search-headless": "2.0.0-alpha.134",
```

Comment:
why do we need to pin to this specific alpha version?

Reply:
this was addressed per Yen's comment in all-hands.